Index: draw_tool.sln
===================================================================
--- draw_tool.sln	(revision 226)
+++ draw_tool.sln	(working copy)
@@ -1,8 +1,6 @@
 ﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
-MinimumVisualStudioVersion = 10.0.40219.1
+Microsoft Visual Studio Solution File, Format Version 11.00
+# Visual C++ Express 2010
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "draw_tool", "draw_tool\draw_tool.vcxproj", "{3AABEC35-1DC1-46FA-948F-1B27A2C0897B}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "unit_test", "unit_test\unit_test.vcxproj", "{A8197525-AB86-4645-AA53-87B3569146E6}"
Index: draw_tool/draw_tool.vcxproj
===================================================================
--- draw_tool/draw_tool.vcxproj	(revision 226)
+++ draw_tool/draw_tool.vcxproj	(working copy)
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
     <ProjectConfiguration Include="Debug|Win32">
       <Configuration>Debug</Configuration>
@@ -20,7 +20,7 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{3AABEC35-1DC1-46FA-948F-1B27A2C0897B}</ProjectGuid>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>draw_tool</RootNamespace>
   </PropertyGroup>
@@ -30,7 +30,6 @@
     <UseDebugLibraries>true</UseDebugLibraries>
     <CLRSupport>Pure</CLRSupport>
     <CharacterSet>Unicode</CharacterSet>
-    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
@@ -44,7 +43,6 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <CLRSupport>Pure</CLRSupport>
     <CharacterSet>Unicode</CharacterSet>
-    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
Index: draw_tool/draw_tool.vcxproj.filters
===================================================================
--- draw_tool/draw_tool.vcxproj.filters	(revision 226)
+++ draw_tool/draw_tool.vcxproj.filters	(working copy)
@@ -89,9 +89,6 @@
     <ClInclude Include="..\..\lib\dt_layer.h">
       <Filter>ヘッダー ファイル\lib</Filter>
     </ClInclude>
-    <ClInclude Include="..\..\lib\dt_group.h">
-      <Filter>ヘッダー ファイル\lib</Filter>
-    </ClInclude>
   </ItemGroup>
   <ItemGroup>
     <ResourceCompile Include="app.rc">
Index: draw_tool/Form1.h
===================================================================
--- draw_tool/Form1.h	(revision 226)
+++ draw_tool/Form1.h	(working copy)
@@ -477,7 +477,7 @@
 #pragma endregion
 
 		void	proc_tool_buttons(const string&name) {
-			const	map<string, int>		names = {
+			const	map<string, int>		names/* = {
 				{ "Select", 0 },
 				{ "Direct Select", 1 },
 				{ "Anchor Point", 2 },
@@ -493,7 +493,7 @@
 				{ "Zoom", 12 },
 				{ "Hand", 13 },
 				{ "Scissors", 14 },
-			};
+			}*/;
 			cli::array<System::Windows::Forms::Button^>^	buttons = {
 				buttonSelect,
 				buttonDirectSelect,
Index: unit_test/unit_test.vcxproj
===================================================================
--- unit_test/unit_test.vcxproj	(revision 226)
+++ unit_test/unit_test.vcxproj	(working copy)
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
     <ProjectConfiguration Include="Debug|Win32">
       <Configuration>Debug</Configuration>
@@ -20,7 +20,7 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{A8197525-AB86-4645-AA53-87B3569146E6}</ProjectGuid>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>unit_test</RootNamespace>
   </PropertyGroup>
@@ -30,7 +30,6 @@
     <UseDebugLibraries>true</UseDebugLibraries>
     <CLRSupport>true</CLRSupport>
     <CharacterSet>Unicode</CharacterSet>
-    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
@@ -44,7 +43,6 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <CLRSupport>true</CLRSupport>
     <CharacterSet>Unicode</CharacterSet>
-    <PlatformToolset>v140</PlatformToolset>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
Index: unit_test/unit_test.vcxproj.filters
===================================================================
--- unit_test/unit_test.vcxproj.filters	(revision 226)
+++ unit_test/unit_test.vcxproj.filters	(working copy)
@@ -163,12 +163,6 @@
     <ClCompile Include="..\..\unit_test\unit_test_main.cpp">
       <Filter>ソース ファイル\unit_test</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\unit_test\dt_path\test_path_to_postscript.cpp">
-      <Filter>ソース ファイル\unit_test\dt_path</Filter>
-    </ClCompile>
-    <ClCompile Include="..\..\lib\dt_group.cpp">
-      <Filter>ソース ファイル\lib</Filter>
-    </ClCompile>
     <ClCompile Include="..\..\lib\dt_obj.cpp">
       <Filter>ソース ファイル\lib</Filter>
     </ClCompile>
